packages/tests/server/___testHelpers.ts

```diff
@@ -117,9 +119,21 @@ export function routerToServerAndClientNew<TRouter extends AnyRouter>(
   });
   const trpcClientOptions = {
     links: [
-      httpBatchLink({
-        url: httpUrl,
-        transformer: opts?.transformer as any,
+      splitLink({
+        condition(op) {
+          // check for context property `skipBatch`
+          return Boolean(op.context['skipBatch']);
```

Comment:
This is a neat solution, but I'd love to see another. Maybe we generate new proxy methods when a procedure is a non-json type?

Ie for the new client: `path.to.foo.blobQueryOptions({ })`